packages/aws-cdk-lib/aws-stepfunctions/README.md

```diff
@@ -124,6 +124,7 @@ becomes new the new Data object. This behavior can be modified by supplying valu
 
 These properties impact how each individual step interacts with the state machine data:
 
+* `stateName`: the custom state name to each state if required else we used the construct id
```

Comment:
"if required"

Required by who?

Reply:
Please use punctuation as well.

"the name of the state in the state machine definition. If not supplied, defaults to the construct id."

Or something.